crates/verify/src/bytecode.rs

```diff
@@ -199,10 +199,13 @@ impl VerifyBytecodeArgs {
                 creation_data.transaction_hash
             );
         };
+
         // Extract creation code
-        let maybe_creation_code = if receipt.contract_address == Some(self.address) {
+        let maybe_creation_code = if receipt.to == Some(Address::ZERO) &&
```

Comment:
shouldn't this also include `receipt.to.is_none()`?

Reply:
thanks